lm_eval/tasks/schemingtasks/include_task_folder.py

```diff
@@ -0,0 +1,2 @@
+from lm_eval.tasks import include_task_folder
+include_task_folder("/Users/marina.levay/Documents/dioptra/lm-evaluation-harness/lm_eval/tasks/scheming-tasks")
```

Comment:
currently references local files, and .DS_STORE files should not be committed

Reply:
Thank you for pointing out! It was a mistake.